src/CoreWCF.Metadata/tests/.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
Okay, I see why now. You could also use this mechanism on the test which required modifying the connection limit for .NET Framework.

Reply:
I introduced XunitTestAssemblyAttribute.NoConcurrency.cs to disable concurrency. Inclusion in this project is controlled by Directory.Build.* files